verify_test_type/README.md

```diff
@@ -0,0 +1,43 @@
+<h1 align="center">Verify Test Type</h1>
```

Comment:
Why use HTML tags in a Markdown?

Reply:
Since this was used for certain scripts in idoft for example in https://github.com/TestingResearchIllinois/idoft/blob/main/format_checker/README.md. Tried to make README consistent.

Reply:
Change those `README`s (likely in a different PR).

Reply:
So should i remove the html tags or not from the markdown in this README file?

Reply:
Yes, remove (rather replace with `#` and such) in this file in this PR. Then open another PR to remove from the other README files.